run.go

```diff
@@ -13,8 +13,13 @@ func newRunCmd() *cobra.Command {
 		Use:   "run",
 		Short: "Run and hold kloader",
 		Run: func(cmd *cobra.Command, args []string) {
-			mounter := NewConfigMapMounter(getRestConfig(), configMap, mountDir, bashFile)
-			mounter.Run()
+			if configMap != "" {
+				mounter := NewConfigMapMounter(getRestConfig(), configMap, mountDir, bashFile)
+				mounter.Run()
+			} else if secret != "" {
```

Comment:
If's a separate condition. They are not either or.

Reply:
I told him to make it either or. I think that makes sense, since the actual command is going to be same.